packages/block-library/src/video/tracks-editor.js

```diff
@@ -177,8 +175,7 @@ function SingleTrackEditor( { track, onChange, onClose, onRemove } ) {
 							{ __( 'Close' ) }
 						</Button>
 						<Button
-							// TODO: Switch to `true` (40px size) if possible
-							__next40pxDefaultSize={ false }
+							__next40pxDefaultSize
```

Comment:
This can be seen while you add the core/video and add some track in track list.

**Note:** This would have no impact, as it have link style for the button. Still as a part of to make button consistent, we have added this change.

| Before | After     |
| :-------- | :------- |
| <img width="404" alt="Remove-track-before" src="https://github.com/user-attachments/assets/4b819019-3f84-4abc-932f-aa9d4f219423"> | <img width="616" alt="remove-track-after" src="https://github.com/user-attachments/assets/a3d5ef45-7f0e-4162-83da-e00b57f9ff82"> |